app/design/frontend/Diana/default-theme/web/js/image-zoom-widget.js

```diff
@@ -0,0 +1,22 @@
+define(['jquery', 'jquery/ui'], function($) {
+    'use strict';
+
+    $.widget('mage.imagezoom', {
+
+        _create: function () {
+            console.log('image-zoom-widget created');
+            this.element
+                .delegate("img", "click", function(element) {
+                    if(element.currentTarget.style.zoom == '150%') {
+                        $(element.currentTarget).css('zoom', 100 + '%');
+                        $(element.currentTarget).siblings().css('zoom', 100 + '%')
```

Comment:
@Dianaeva Please, change code to use css classes. You should not change styling via JS